pkg/nodebootstrap/userdata.go

```diff
@@ -96,20 +98,35 @@ func clusterDNS(spec *api.ClusterConfig, ng *api.NodeGroup) string {
 	return "10.100.0.10"
 }
 
-func makeKubeletParamsCommon(spec *api.ClusterConfig, ng *api.NodeGroup) []string {
+func makeKubeletConfigYAML(spec *api.ClusterConfig, ng *api.NodeGroup) ([]byte, error) {
+	data, err := Asset("kubelet.yaml")
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: use "k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig" and runtime.Decode/Encode
```

Comment:
Current approach is okay, but probably best to try switching over to using the struct.